satnogsclient/observer/orbital.py

```diff
@@ -1,56 +1,106 @@
 # -*- coding: utf-8 -*-
-import sys
-
 from datetime import datetime
-
-import ephem
 import pytz
+import ephem
```

Comment:
Why are you changing the order. We generally want them alphabetically ordered (not required from PEP8 though).